README.md

```diff
@@ -218,14 +218,37 @@ For example, the following will create the `/opt/myapps/hello2.dockerapp` folder
 $ docker-app fork remote/hello.dockerapp:1.0.0 mine/hello2 --path /opt/myapps
 ```
 
+## Storing images in the application package
+
+Using the `docker-app image add` command, one can store all images used by the
+application into the `images` subfolder.
```

Comment:
With the new `attachments` feature, it means the images will be included in the app image as-is. Do we want that? Or maybe we need to ignore this directory?